tdvt/tdvt/tdvt.py

```diff
@@ -56,7 +56,7 @@ def copy_output_file(c, src_name, src_dir):
             return
 
     @classmethod
-    def write_test_results_csv(c):
+    def write_test_results_csv(c, custom_output_dir=''):
```

Comment:
```suggestion
    def write_test_results_csv(c, custom_output_dir: str=''):
```
I _think_ that's the right format for this type hint.